ios-drag-drop.js

```diff
@@ -136,8 +136,10 @@
       var touch = event.changedTouches[0];
       var x = touch[coordinateSystemForElementFromPoint + 'X'];
       var y = touch[coordinateSystemForElementFromPoint + 'Y'];
-      dropEvt.offsetX = x - target.x;
-      dropEvt.offsetY = y - target.y;
+      dropEvt.clientX = x;
+	  dropEvt.clientY = y;
+	  dropEvt.offsetX = x - event.target.clientLeft;
+	  dropEvt.offsetY = y - event.target.clientTop;
```

Comment:
If ```event.target.clientLeft``` here can be replaced with ```target.clientLeft``` then I think we should do that.

I haven't looked too closely at how (if at all) the targets differ here, but I do see that we spend some considerable effort deriving ```target``` via ```elementFromTouchEvent```.